vividus-plugin-mobile-app/src/main/resources/spring.xml

```diff
@@ -9,13 +9,17 @@
      http://www.springframework.org/schema/context https://www.springframework.org/schema/context/spring-context.xsd"
      default-lazy-init="true" profile="mobile_app">
 
-     <context:component-scan base-package="org.vividus.bdd.mobileapp.variable"/>
+     <bean id="clipboard-text" class="org.vividus.bdd.mobileapp.variable.ClipboardTextDynamicVariable">
+         <constructor-arg index="0" value="${mobile-environment.real-device}" />
```

Comment:
we need a new class (e.g. `MobileEnvironment`) to avoid injection of one property into different beans and to avoid passing a bulk of properties